tests/file_storage/tests.py

```diff
@@ -545,7 +553,7 @@ def get_available_name(self, name, max_length=None):
         """
         Append numbers to duplicate files rather than underscores, like Trac.
         """
-        basename, *ext = name.split('.')
+        basename, *ext = str(name).split('.')
```

Comment:
I would use `os.path.splitext()`, e.g.
```python
    def get_available_name(self, name, max_length=None):
        """
        Append numbers to duplicate files rather than underscores, like Trac.
        """
        basename, *ext = os.path.splitext(name)
        number = 2
        while self.exists(name):
            name = ''.join([basename, '.', str(number)] + ext)
            number += 1

        return name
```